### PR TITLE
Example of consuming merge-ci artifacts

### PR DESCRIPTION
**Pull request by manics**
Seeing as this is intended for developers I thought it'd be useful to show how to consume a CI artifact from Nexus.

### CODE REVIEW

**Comment by joshmoore** [comment]
Are there any barriers to having the nexus port open via web-proxy?

**Comment by manics** [comment]
Not sure. Nexus logins should be blocked https://github.com/ome/devspace/blob/86f4ed9b11e52950b2fdd099e5a1ef916a5829ef/jenkins/conf.d/jenkins.conf#L42-L45
but if we start using it for other repos (e.g. Docker registry?) I don't know if additional config would be needed. It might be sufficient to limit external nexus requests to `GET HEAD` only. This doesn't work for the nexus UI, but it might work for maven API calls.

File: README.md [hunk 1]
```diff
@@ -22,3 +22,51 @@ Install [Gradle](https://gradle.org/), and build this project:
     ./minimal-omero-client-5.4.10/bin/minimal-omero-client --omero.host=localhost --omero.user=USERNAME --omero.pass=PASSWORD
 
 You can pass additional properties, e.g. to debug SSL issues pass `--IceSSL.Trace.Security=1`.
+
+# Using development CI libraries
+
+Gradle will not accept a self-signed HTTPS certificate so you must first setup a local proxy.
```
Comment (mtbc) [inline README.md]: set up

**Comment by manics** [comment]
Updated:
- Use latestci instead of mergeci
- Use the public web proxy

**Pull request merged**